include/fs/littlefs.h

```diff
@@ -12,6 +12,7 @@
 #include <storage/flash_map.h>
 
 #include <lfs.h>
+#include <fs/fs.h>
```

Comment:
Why is the header needed here?

Reply:
Without including `<fs/fs.h>` following errors pops up:
```
 error: unknown type name 'fs_mode_t'; did you mean 'sys_snode_t'? 
 error: 'const struct fs_file_system_t' has no member named 'open'
```

when I build the `littlefs_blk` sample program.